src/08_comprehensions.py

```diff
@@ -11,14 +11,16 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
```

Comment:
comprehensions are one liners, see if you can do something like: y = [word.upper() for word in a]